From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: base-BUILD.gn / com

Files:
 base/BUILD.gn
 build/config/win/BUILD.gn

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/base/BUILD.gn b/base/BUILD.gn
index b2c4562aa4536..acec47a0be298 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -1232,6 +1232,11 @@ component("base") {
       "/DELAYLOAD:cfgmgr32.dll",
       "/DELAYLOAD:powrprof.dll",
       "/DELAYLOAD:setupapi.dll",
+      "/DELAYLOAD:dbghelp.dll",
+      "/DELAYLOAD:dwmapi.dll",
+      "/DELAYLOAD:shell32.dll",
+      "/DELAYLOAD:uxtheme.dll",
+      "/DELAYLOAD:propsys.dll",
     ]
   }
 }
diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
index edb193a5f3ac0..614f8e0297940 100644
--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -401,7 +401,6 @@ config("delayloads") {
     "/DELAYLOAD:oleaut32.dll",
     "/DELAYLOAD:pdh.dll",
     "/DELAYLOAD:propsys.dll",
-    "/DELAYLOAD:psapi.dll",
     "/DELAYLOAD:rpcrt4.dll",
     "/DELAYLOAD:rstrtmgr.dll",
     "/DELAYLOAD:secur32.dll",

